docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -65,22 +66,22 @@ export default class ComponentExample extends Component {
               >
                 {this.props.title}
                 <a href={`#${this.anchor}`}>
-                  <Icon className='linkify' style={linkIconStyle} />
+                  <Icon name='linkify' style={linkIconStyle} />
                 </a>
               </h3>
               <p>{this.props.description}</p>
             </Grid.Column>
             <Grid.Column width={4} className='right aligned'>
               <Button className='basic mini labeled icon' onClick={this.toggleShowCode}>
                 code
-                <Icon className='code' />
+                <Icon name='code' />
               </Button>
             </Grid.Column>
           </Grid>
         </Grid.Column>
         {this.props.children && children}
         <Grid.Column>
-          {createElement(this.component)}
+          {createElement(this.component, this.props)}
```

Comment:
It looks like it is adding the ComponentExample's props `children` `description` `examplePath` `exampleSrc` and `title` onto the rendered component example.  I don't see any examples using these props and some, like children, could have unintended consequences.

If this is meant to pass down `category={category}`, we can use the `getUnhandledProps()` util to grab any unknown/extra props and use those here instead.